backend/src/rest-api-docs.yaml

```diff
@@ -287,6 +306,50 @@ components:
           type: string
         emailAddress:
           type: string
+    PostMatchRequestBody:
+      type: object
+      properties:
+        matchId:
+          description: Identifier of the match that is supposed to start
+          type: string
+          format: uuid
+          example: 2480fc2c-bbd7-427c-9439-02f76583aef5
+        firstMatchSideId:
+          description: Identifier of the match first team
+          type: string
+          format: uuid
+          example: 3510fc2c-bbd7-427c-9439-02f76583aef5
+        secondMatchSideId:
+          description: Identifier of the match second team
+          type: string
+          format: uuid
+          example: 8321fc2c-bbd7-427c-9439-02f76583aef7
+      required:
+        - tournamentId
+        - firstMatchSideId
+        - secondMatchSideId
+    PostMatchResponseBody:
```

Comment:
Ten response jest całokowicie zbędny :) Wszystko klient już zna, bo to wysyła wcześniej, więc w odpowiedzi nic nie jest potrzebne, a wręcz nie wskazane. Dodatkowy transfer i odpowiedź sprawia, że niby powinienem ją analizować, a tutaj zawsze by była ignorowana.

Reply:
przyznaję się bez bicia, przekleiłem to bez zastanowienia, już usuwam :)